### ISSUE TITLE
[Bug]: Incorrectly setting time created as time modified

### ISSUE THREAD

**Issue by trevoracles-code** [remotely-save/remotely-save#769]
### What happened?

Steps:

1. System A: Create a file on one system, hit sync. Let's say the created time is 13:00.
2. System B: Sync, modify the file, then sync (modified time is 13:05, created is 13:00).
3. System A:  Sync again. Now the modified _and_ created time on A are both set to the modified time from B (13:05).

This happens if A or B are both Windows, or one is Windows and the other Android. Could be wrong, but I think the root cause is below where you are sending mtimeCli twice to writeFile() where you have a "TODO" item in copyLogic.ts. 

Seems like you would want to _not_ include a parameter for ctime when writing a modification mixed entity, but you would want it for any created_then_pull actions?

[(copyLogic.ts)](https://github.com/remotely-save/remotely-save/blob/52f3e6a265f940664dd0cebe1ad4b43424577048/src/copyLogic.ts#L44)
```typescript
  // console.debug(`copyFile: about to start right.writeFile`);
  return {
    entity: await right.writeFile(
      key,
      content,
      statsLeft.mtimeCli,
      statsLeft.mtimeCli /* TODO */
    ),
    content: content,
  };
``` 

### What OS are you using?

Windows, Android

### What remote cloud services are you using? (Please choose the specified one if it's in the list)

OneDrive for personal

### Version of the plugin

0.5.16

### Version of Obsidian

1.6.7

### Using password or not

- [ ] Yes.

### Ensure no sensitive information

- [X] I ensure that no sensitive information is submitted in the issue.

**Comment by fyears** [comment]
unfortunately only onedrive supports preserving ctime so initially i didn't implement it.

i will try to implement it when i have free time in next version

**Comment by fyears** [comment]
released on 0.5.19
